manifests/t/TCHelicon/Driver/5.57/TCHelicon.Driver.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TCHelicon.Driver
+PackageVersion: '5.57'
+PackageLocale: en-US
+Publisher: TC Helicon

```

```suggestion
Publisher: TC-Helicon
```
Should match Control Panel